controller.js

```diff
@@ -63,9 +63,8 @@ exports.deleteComments = (req, res, next) => {
 }
 
 
-
-
-
-
-
-
+exports.getUsers = (req, res, next) => {
+    selectUsers().then(({rows}) => {
+        res.status(200).send({users: rows})
+    })
```

Comment:
this should have a catch block on it just in case!

Reply:
Done